docs/blog/2023-05-11_PHP_configuration.md

```diff
@@ -102,7 +102,7 @@ Last, but not least, you can also plug in any extension you write like custom co
 To tie it all together, you have to specify that you want deptrac to you your php config file, for example like this:
 
 ```console
-$ php deptrac.php -c deptrac.config.php
+$ php deptrac.php -c deptrac.php
```

Comment:
This should not work? `deptrac.php` being the executable as well as the config file?

Reply:
You are right. The deptrac bin is not called "deptrac.php" and second "-c deptrac.php" should not be necessary.